assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -682,12 +685,93 @@ const baseSelectors = {
 				return undefined;
 			}
 
-			const audiencesUserCountReportOptions =
-				getAudiencesUserCountReportOptions( configurableAudiences );
+			// eslint-disable-next-line @wordpress/no-unused-vars-before-return
+			const [ siteKitAudiences, otherAudiences ] =
+				getConfiguredSiteKitAndOtherAudiences();
+
+			const isSiteKitAudiencePartialData =
+				hasAudiencePartialData( siteKitAudiences );
+
+			if ( undefined === isSiteKitAudiencePartialData ) {
+				return undefined;
+			}
+
+			const siteKitUserCountReportError = isSiteKitAudiencePartialData
+				? getErrorForSelector( 'getReport', [
+						getSiteKitAudiencesUserCountReportOptions(),
+				  ] )
+				: undefined;
+
+			const otherUserCountReportError =
+				! isSiteKitAudiencePartialData || otherAudiences?.length !== 0
+					? getErrorForSelector( 'getReport', [
+							getAudiencesUserCountReportOptions(
+								isSiteKitAudiencePartialData
+									? otherAudiences
+									: configurableAudiences
+							),
+					  ] )
+					: undefined;
+
+			return [ otherUserCountReportError, siteKitUserCountReportError ];
+		}
+	),
+
+	getSiteKitAudiencesUserCountReportOptions: createRegistrySelector(
+		( select ) =>
+			( state, { startDate, endDate } = {} ) => {
```

Comment:
It doesn't look like the `startDate` and `endDate` options are being used anywhere, so let's remove them to keep things simpler.

```suggestion
			( state ) => {
```